data/web/lang/lang.de.php

```diff
@@ -340,6 +340,8 @@
 $lang['edit']['backup_mx_options'] = 'Backup MX Optionen:';
 $lang['edit']['relay_domain'] = 'Relay Domain';
 $lang['edit']['relay_all'] = 'Alle Empfänger-Adressen relayen';
+$lang['edit']['relay_sender'] = 'Alle gesendete Mails an den Relayhost relayen';
+$lang['edit']['relay_recipient'] = 'Alle empfangene Mails an den Relayhost relayen';
```

Comment:
I'd rather use:
- Alle gesendeten E-Mails an den Relayhost weiterleiten
- Alle empfangenen E-Mails an den Relayhost weiterleiten

Reply:
Sound's better, you're right.